services/docker-compose.devel.yml

```diff
@@ -50,6 +50,10 @@ services:
     volumes:
       - ./dynamic-scheduler:/devel/services/dynamic-scheduler
       - ../packages:/devel/packages
+    # NOTE: if the service needs to be debugged, put replicas to 1
+    # or `docker service update master-simcore_dynamic-scheduler --replicas 1`
+    deploy:
+      replicas: 2
```

Comment:
why replicas:2 only in devel mode?
if you want to have it in 2 replicas all the time, you need to have this in docker-compose.local.yml.
I thought for devel it's better to have only 1 replica? what's the point of having 2 here?

Reply:
Great suggestion. I've missed it.
Ok now when you run up-devel it has 1 replica. all other cases 2.